source/find_getmore_killcursors_commands.rst

```diff
@@ -462,7 +479,7 @@ Semantics of maxTimeMS for a Driver
 
 In the case of  a **non-tailable cursor query** OR **a tailable cursor query with awaitData == false**, the driver MUST set maxTimeMS on the **find** command and MUST NOT set maxTimeMS on the **getMore** command.
 
-In the case of **a tailable cursor with awaitData == true** the driver MUST provide a Cursor level option named **maxAwaitTimeMS** (See CRUD specification for details). The **maxTimeMS** option on the **getMore** command MUST be set to the value of the option **maxAwaitTimeMS**. If no **maxAwaitTimeMS** is specified, the driver SHOULD not set **maxTimeMS** on the **getMore** command. 
+In the case of **a tailable cursor with awaitData == true** the driver MUST provide a Cursor level option named **maxAwaitTimeMS** (See CRUD specification for details). The **maxTimeMS** option on the **getMore** command MUST be set to the value of the option **maxAwaitTimeMS**. If no **maxAwaitTimeMS** is specified, the driver SHOULD not set **maxTimeMS** on the **getMore** command.
```

Comment:
"cursor-level" would seem consistent with earlier references to "server-side". I realize it's in the original text, so no objections if you want to leave as-is.

Optionally link to the CRUD spec, since we already have an anchor defined from an earlier paragraph. "See" can also be made lowercase.

Edit: Just realized the only change here was removing trailing whitespace. Feel free to ignore my suggestions.

Reply:
Yes, that's my IDE automatically trimming trailing whitespace. I can remove the change from the diff and take care of this in the separate PR that brings in formatting.